graphics/src/AssimpLoader.cc

```diff
@@ -177,16 +182,50 @@ void AssimpLoader::Implementation::RecursiveCreate(const aiScene* _scene, const
   }
 }
 
+void AssimpLoader::Implementation::RecursiveStoreBoneNames(const aiScene *_scene, const aiNode *_node)
```

Comment:
I guess I can see how it would be nicer (but significantly more work, can probably postpone to a "if we have time") to have the new skeleton container in assimp to be populated also in Collada (or even GLB) assets

Reply:
I will put "Use the new skeleton structure" to my TO-DO list. This will remove the need for this func.

Reply:
I saw positive feedback on your PR in `assimp` on this, I wonder if we can make it happen "soon enough"